nrf_security/doc/backend_config.rst

```diff
@@ -82,44 +52,44 @@ Feature support
 |             |                   | 256-bit key |
 +-------------+-------------------+-------------+
 
+.. note::
+   * The :ref:`nrf_security_backends_oberon` uses some functionality from Original Mbed TLS for AES operations.
+
```

Comment:
We can remove the "Original Mbed TLS" row from this table.

Users will be enabling nrf_oberon and see from the nrf_oberon row which key sizes are supported.

Reply:
The same applies elsewhere.

Reply:
Original Mbed TLS rows are removed